autoload/lumen/platforms/macos.vim

```diff
@@ -1,22 +1,23 @@
-let s:path = resolve(expand('<sfile>:p:h')) . '/macos/watcher'
+let s:exe = resolve(expand('<sfile>:p:h')) . '/macos/watcher'
 
 func lumen#platforms#macos#watch_cmd()
-	return [s:path . ".swift"]
+	return [s:exe]
 endfunc
 
 func lumen#platforms#macos#parse_line(line)
-	if a:line == "LDark"
+	if a:line =~ "^Dark_.*"
 		call lumen#dark_hook()
 	else
 		call lumen#light_hook()
 	endif
 endfunc
 
 func lumen#platforms#macos#oneshot()
-	let out = system(printf("%s get || swiftc %s.swift", s:path, s:path))
+	let out = system(printf("%s get || swiftc %s.swift -o %s",s:exe,s:exe,s:exe))
 	if len(out) != 6
+		echom v:shell_error
```

Comment:
I think you accidentally left this line there from debugging. This plugin should fallback silently, e.g. now it would output an error on first run after install even though everything is fine, so please remove this line.